packages/web-components/src/components/va-number-input/va-number-input.tsx

```diff
@@ -82,6 +82,11 @@ export class VaNumberInput {
    */
   @Prop({ mutable: true, reflect: true }) value?: string;
 
+  /**
+   * Is this input used for a currency value?
```

Comment:
Could this be a statement instead of a question? (to keep it consistent with the other prop descriptions)

Reply:
done!